src/devicecontroller/DefaultDeviceController.ts

```diff
@@ -799,6 +799,11 @@ export default class DefaultDeviceController implements DeviceControllerBasedMed
     device: Device,
     fromAcquire: boolean
   ): Promise<void> {
+    if (device === undefined) {
```

Comment:
The 2.0 device controller runs more operations after calling `chooseInputIntrinsicDevice()`. For example, if the `device` argument is `undefined`, it'll call `pushVideoMeetingStateForPermissions()`.

https://github.com/aws/amazon-chime-sdk-js/blob/bfc4c600fb7e68f2d358ecb6c7fd096d30b2d430/src/devicecontroller/DefaultDeviceController.ts#L177-L185

Then, the `undefined` device will be considered as `videoInputSelected`.

https://github.com/aws/amazon-chime-sdk-js/blob/bfc4c600fb7e68f2d358ecb6c7fd096d30b2d430/src/devicecontroller/DefaultDeviceController.ts#L129-L134

---

- Please make sure that just returning void for the `undefined` device makes sense in other cases.

- How about throwing an error?